ddtrace/contrib/django/utils.py

```diff
@@ -32,44 +32,28 @@ def quantize_key_values(key):
     return key
 
 
-def get_request_uri(request):
-    """
-    Helper to rebuild the original request url
+def get_django_2_route(resolver, resolver_match):
+    # Try to use `resolver_match.route` if available
+    # Otherwise, look for `resolver.pattern.regex.pattern`
+    route = resolver_match.route
+    if not route:
+        # DEV: Use all these `getattr`s to protect against changes between versions
```

Comment:
Since it is just a matter of one of these attributes not being present for the lookup to fail, it can be simpler just to wrap this in a `try`/`except` that catches an `AttribueError`.